pcsx2/DEV9/ATA/ATA_Info.cpp

```diff
@@ -247,28 +260,35 @@ void ATA::CreateHDDinfo(u64 sizeSectors)
 	 * bit 13: IDLE IMMEDIATE with UNLOAD FEATURE
 	 * bit 14: 1
 	 */
-	WriteUInt16(identifyData, &index, static_cast<u16>((1 << 14) | (1 << 1) | 1)); //word 84
+	WriteUInt16(identifyData, &index, ( 	//word 84
+		(1 << 0) |							//implemented
+		(1 << 1) | 							//implemented
+		(1 << 14)));						//Fixed
 	//Command set/feature enabled/supported (See word 82)
-	WriteUInt16(identifyData, &index, static_cast<u16>((fetSmartEnabled << 0) | (fetSecurityEnabled << 1) | (fetWriteCacheEnabled << 5) | (fetHostProtectedAreaEnabled << 10) | (1 << 14))); //Fixed      //word 85
+	WriteUInt16(identifyData, &index, ( 	//word 85
+		(fetSmartEnabled << 0) | 			//implemented
+		(fetSecurityEnabled << 1) | 		//not implemented
+		(fetWriteCacheEnabled << 5) | 		//implemented
+		(fetHostProtectedAreaEnabled << 10)|//not implemented
+		(1 << 14)));						//Fixed
 	//Command set/feature enabled/supported (See word 83)
-	// clang-format off
-	WriteUInt16(identifyData, &index, static_cast<u16>(
+	WriteUInt16(identifyData, &index, ( 	//word 86
 		/*(1 << 8) |						//SET MAX */
-		((lba48Supported ? 1 : 0) << 10) |	//Fixed
+		(lba48Supported << 10) |			//user defined
 		(1 << 12) |							//Fixed
-		(1 << 13)));						//Fixed      //word 86
+		(1 << 13)));						//Fixed
```

Comment:
Where I've used `Fixed` I'm referring to what the spec notes the field as (variable or fixed)

Following that convention, the LBA 48 field would count as `Fixed`. even if it is user defined

Reply:
Fixed